frontend/src/app/order/edit-document/edit-document.component.spec.ts

```diff
@@ -11,14 +11,15 @@ import { provideI18nTesting } from '../../i18n/mock-18n.spec';
 import { provideRouter } from '@angular/router';
 
 @Component({
-  template: '<rb-edit-document [order]="order" [uploadProgress]="progress" (saved)="saved = $event" (cancelled)="cancelled = true" />',
-  imports: [EditDocumentComponent]
+  template: '<rb-edit-document [order]="order()" [uploadProgress]="progress()" (saved)="saved = $event" (cancelled)="cancelled = true" />',
+  imports: [EditDocumentComponent],
+  changeDetection: ChangeDetectionStrategy.OnPush
 })
 class TestComponent {
-  order = {
+  readonly order = signal({
     documents: []
-  } as DetailedOrder;
-  progress: number | null = null;
+  } as DetailedOrder);
+  readonly progress = signal<number | null>(null);
   cancelled = false;
   saved: DocumentCommand = null;
```

Comment:
nit: these should be signals too

Reply:
I only use them in the tests. They're not displayed nor passed to anything.

Reply:
I agree it works but I don't like to make the distinction. I'll change them